models/marts/advertising/fct_ad_performance.sql

```diff
@@ -47,14 +47,19 @@ marketing_channels as (
     select
     
         *,
-        case
-
+        case 
             when lower(campaign) ilike '%display%' then 'display'
             when lower(campaign) ilike '%shopping%'
-                or lower(campaign) ilike '%cpc%' then 'paid search' 
-            when utm_medium is null then 'direct'
-            when lower(utm_medium) in ('retargeting', 'social', 'influencers') then utm_medium
-            when lower(utm_medium) in ('paid', 'paid_social') then 'facebook ads'
+                or lower(campaign) ilike '%cpc%' then 'paid search'
+            when utm_medium is null and utm_medium not in ('fb_retargeted', 'ig_retargeted') 
```

Comment:
the logic here doesn't make sense to me. something cannot be null and be `in` something. is this supposed to be an OR statement?

Reply:
oops, this was supposed to be `... and utm_campaign...` not  `and utm_medium`